auto-completion-controller

From: Bryan Larsen <bryan@larsen.st>

## A form with auto-completion

Now we'll create the form to add a new person to the project. We'll set it up so that you can type the user's name, with auto-completion, in order to add someone to the project.

First we need the controller side of the auto-complete. Add this declaration to `projects_controller.rb`:

SHOW_PATCH
---

 app/controllers/users_controller.rb |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)


diff --git a/app/controllers/users_controller.rb b/app/controllers/users_controller.rb
index 4839bd0..42d49df 100644
--- a/app/controllers/users_controller.rb
+++ b/app/controllers/users_controller.rb
@@ -4,6 +4,8 @@ class UsersController < ApplicationController
 
   auto_actions :all, :except => [ :index, :new, :create ]
 
+  autocomplete
+
   def do_signup
     hobo_do_signup do
       if errors.blank?
